doc/conf.py

```diff
@@ -282,11 +283,17 @@ def setup(app):
 
 # -- Options for readthedocs ----------------------------------------------
 
-
 # -- Copy documentation for Microcircuit Model ----------------------------
 
-source = r'../pynest/examples/Potjans_2014'
-destination = r'examples'
-
-if os.path.exists(destination) and os.path.isdir(destination):
-    copytree(source, destination, ignore=ignore_patterns('*.dat', '*.py', '*.rst'), dirs_exist_ok=True)
+source = Path("../pynest/examples/Potjans_2014")
+destination = Path("examples")
+
+# copy all *.png with the right subfolder from source to destination
+for f in source.rglob('*.png'):
+    # ff - files with subfolder
+    ff = destination / Path(*f.parts[4:])
+    # fp - only the path, without file name
+    fp = destination / Path(*f.parts[4:]).parent
+    if not fp.exists():
+        Path(fp).mkdir(parents=True, exist_ok=True)
+    copyfile(f, ff)
```

Comment:
```suggestion
def copytreeglob(source, target, glob='*.png'):
    ''' 
    Recursively copy all files selected by `glob` from `source` to `target` path,
    recreating the sub-folder structure.

    Parameters
    ----------
    source : path, str
        source folder where to recursively search for glob

    target : path, str
        target folder where to recreate the tree of source files

    glob : str
        shell-glob specifying which files to copy
    '''
    source = Path(source)
    target = Path(target)
    for relativename in [x.relative_to(source) for x in source.rglob(glob)]:
        # manually create directory, since shutil.copyfile() does not support
        # the `dirs_exist_ok=True` below Python-3.8
        targetpath = target/relativename.parents[0]
        if not targetpath.exists():
            targetpath.mkdir(parents=True)
        assert targetpath.is_dir(), "Targetpath is obstructed by a non-directory object (maybe a file)"
        copyfile(source/relativename, target/relativename)


copytreeglob("../pynest/examples/Potjans_2014", "examples", '*.png')

```
This seems a bit longer, but it's mostly the docstring. The for loop is much more readable like this, imho. What do you think?

Reply:
Thank you very much for your comments. I have adopted and tested it.